packages/ntcore-ts-client/src/lib/socket/socket.ts

```diff
@@ -25,6 +25,12 @@ export class NetworkTablesSocket {
   private offset = 0;
   private bestRtt = -1;
 
+  private RECONNECT_TIMEOUT = 1000;
+  private RTT_PERIOD_V4_0 = 1000;
+  private RTT_PERIOD_V4_1 = 250;
+  private TIMEOUT_V4_0 = 3000;
```

Comment:
This is temporarily unused, as v4.0 doesn't have the same timeout logic. If we wanted to add it, 3000ms is probably an ideal amount.